src/reducers/poiForm.reducer.js

```diff
@@ -54,6 +57,18 @@ export default function poiForm(state = initialState.poiForm, action) {
         ...state,
         media: [...state.media].filter(media => media.id !== action.payload.id)
       }
+    case POI_COPIED:
+    case POI_FORM_POI_PASTED:
+      var newClipboard = [...state.clipboard].filter(
+        poi => poi.id !== action.payload.id
+      )
+      if (newClipboard.length === POI_FORM_MAX_CLIPBOARD_LENGTH)
```

Comment:
intuitively, we shouldn't need to worry about dealing with the max clipboard length when we paste. we only need to worry about it when we copy. so this shouldn't be here

Reply:
the reason I wrote it like that is to reorder the most recent poi to the top of the list. so it's an extra step for the paste but it saves all the lines of code that we would write for copy that only differ by one line.